Tests.Integration.Presentation.Web/Tools/UserHelper.cs

```diff
@@ -37,5 +40,30 @@ public static async Task<HttpResponseMessage> SendGetUsersWithCrossAccessAsync(C
             var cookie = optionalLogin ?? await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
             return await HttpApi.GetWithCookieAsync(TestEnvironment.CreateUrl("api/user/with-cross-organization-permissions"), cookie);
         }
+
+        public static User GetUserByIdWithRightsAsync(int id)
+        {
+            var user = DatabaseAccess.MapFromEntitySet<User, User>(x => x.AsQueryable()
+                .Include(user => user.DataProcessingRegistrationRights)
+                .Include(user => user.OrganizationRights)
+                .Include(user => user.ItContractRights)
+                .Include(user => user.ItProjectRights)
+                .Include(user => user.ItSystemRights)
+                .Include(user => user.OrganizationUnitRights)
+                .Include(user => user.SsoIdentities)
+                .Include(user => user.ItProjectStatuses)
+                .Include(user => user.ResponsibleForCommunications)
+                .Include(user => user.HandoverParticipants)
+                .Include(user => user.ResponsibleForRisks)
```

Comment:
I'm not sure if that's how I'm supposed to include related entities here

Reply:
there is a better way yet not public, we shall do that together